app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :post, foreign_key: 'author_id'
+  has_many :comment
+  has_many :like
+
+  def three_recent_posts
+    Post.where(author: self).order(created_at: :desc).limit(3)
```

Comment:
In the method three_recent_poss tit should be posts not Post as you are targeting to get the five recent comments 👍🏿